<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="THANH37-H15KV3V" failures="3" tests="10" name="Test" time="6.583" errors="0" timestamp="2023-07-06T11:03:13 ICT">
  <testcase classname="demo.Example1Test" name="T01_StatusCodeTest" time="0.019"/>
  <testcase classname="demo.Example1Test" name="T02_IdChecked" time="0.007"/>
  <testcase classname="demo.Example1Test" name="T03_EmailChecked" time="0.01"/>
  <testcase classname="demo.Example1Test" name="T04_FirstNameChecked" time="0.001"/>
  <testcase classname="demo.Example1Test" name="T05_LastNameChecked" time="0.006"/>
  <testcase classname="demo.Example1Test" name="T06_AvatarChecked" time="0.004"/>
  <testcase classname="demo.Example1Test" name="T07_verifyOnMatchingUserId" time="1.259"/>
  <testcase classname="demo.Example2Test" name="T01_StatusCodeTest" time="0.002">
    <failure type="java.lang.AssertionError" message="Status Check Failed! expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Status Check Failed! expected [200] but found [404]
at demo.Example2Test.T01_StatusCodeTest(Example2Test.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- T01_StatusCodeTest -->
  <testcase classname="demo.Example2Test" name="T03_MessageChecked" time="0.002">
    <failure type="java.lang.AssertionError" message="Message field check Failed! expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Message field check Failed! expected [false] but found [true]
at demo.Example2Test.T03_MessageChecked(Example2Test.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- T03_MessageChecked -->
  <testcase classname="demo.Example2Test" name="T03_verifyOnMessageContainName" time="0.032">
    <failure type="java.lang.AssertionError" message="expected [null] but found [User not found!]">
      <![CDATA[java.lang.AssertionError: expected [null] but found [User not found!]
at demo.Example2Test.T03_verifyOnMessageContainName(Example2Test.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- T03_verifyOnMessageContainName -->
</testsuite> <!-- Test -->
